### ISSUE TITLE
Allow find by estimate

### ISSUE THREAD

**Issue by andjosh** [andjosh/clubhouse-cli#21]
It would be nice to use `club find` with an estimate comparator option....

Possible usage:
~~~sh
$ club find --estimate >3 # find with estimate greater than 3
$ club find -e 5 # find with estimate equal to 5
~~~

**Comment by j-martin** [comment]
Should this be closed? https://help.clubhouse.io/hc/en-us/articles/360000046646-Search-Operators#estimate

The only caveat is that you can't have `<, >, >= <=` nor `OR`. So it's pretty limited.

**Comment by andjosh** [comment]
Yeah, that's a big caveat for me. I like seeing `quick wins` under 3 points, etc.

So I'd still like to implement it with proper operator support.
